### ISSUE TITLE
option to set made for kids or not

### ISSUE THREAD

**Issue by robbiedood** [tokland/youtube-upload#300]
Dear tokland,

Youtube & COPPA now requires us to set video made for kids or not. So we were wondering if there is such medata available in the youtoube-upload, e.g. --made_for_kids True | False, for video upload.

Hope to hear your expertise. Thank you.

**Comment by KenSharp** [comment]
The API has hundreds of options.
https://developers.google.com/youtube/v3/docs/videos

**Comment by dev-zero** [comment]
@KenSharp true, but the "made for kids" has a rather prominent place now in the YT UI stating that "Failure to set your content appropriately may result in consequences on YouTube or have legal consequences under COPPA and other laws."

**Comment by justinmarwad** [comment]
Update on this? 

YouTube makes all videos that don't have the "made for kids" option set private. I'm not familiar with this code base, but I can look into helping out. It looks like @mingzeng21 already put down some code that would set the made for kids option? Looks like the other thing left is to make a flag to set the option now.

**Comment by Headstrike** [comment]
Any updates to this?

**Comment by leooxx** [comment]
Hello. I need this options too ! Youtube is not posting the video...
Thank you

**Comment by LaserMoai** [comment]
Your videos get privated for a different reason, see #306.
